src/Rotations.jl

```diff
@@ -13,6 +13,11 @@ generates the $\mathrm{SO}(n) \subset \mathbb R^{n\times n}$
 struct Rotations{N} <: Manifold end
 Rotations(n::Int) = Rotations{n}()
 
+function representation_size(::Rotations{N}, ::Type{T}) where {N,T<:Union{MPoint, TVector, CoTVector}}
```

Comment:
Should return a `Size` for consistency.

Reply:
I think returning a tuple is quite consistent with what `size` returns.

Reply:
That's fine, then it should be consistent everywhere (above and below returns `Size`)